src/server/utilities/cspHeader/index.js

```diff
@@ -104,12 +118,30 @@ const directives = {
       'https://emp.bbc.co.uk',
       'https://chartbeat.com',
       'https://*.chartbeat.com',
+      'https://www.youtube.com', // Social Embeds
+      'https://platform.twitter.com', // Social Embeds
+      'https://www.instagram.com', // Social Embeds
+      'https://syndication.twitter.com', // Social Embeds
       "'self'",
     ],
   },
   imgSrc: {
     ampLive: [
```

Comment:
These have now been added because they're now different to canonical?

Reply:
Yes, that's right. AMP Social Embeds use a different implementation to Canonical.